content/200-concepts/100-components/01-prisma-schema/06-relations.mdx

```diff
@@ -1088,6 +1089,108 @@ model Post {
 }
 ```
 
+## Troubleshooting
+
+Modelling your schema can sometimes offer up some unexpected results. This section aims to cover the most prominent of those.
+
+### How to use a relation table with a many-to-many relationship
+
+There are a couple of ways to define a m-n relationship, implicitly or explicity. Implicitly means letting Prisma handle the relation table (JOIN table) under the hood, all you have to do is define an array/list for the non scalar types on each model, see [implicit many-to-many relations](#implicit-many-to-many-relations).
+
+Where you might run into trouble is when creating an [explicit m-n relationship](#explicit-many-to-many-relations), that is, to create and handle the relation table yourself. **It can be overlooked that Prisma requires both sides of the relation to be present**.
+
+Take the following example, here a relation table is created to act as the JOIN between the `Post` and `Category` tables. This will not work however as the relation table (`PostCategories`) must form a 1-to-many relationship with the other two models respectively.
+
+The back relation fields are missing from the `Post` to `PostCategories` and `Category` to `PostCategories` models.
+
+```prisma
+model Post {
+  id            Int         @id @default(autoincrement())
+  title         String
+  categories    Category[]
+}
+
+model PostCategories {
+  postId        User        @relation(fields: [post], references: [id])
+  post          Post
+  categoryId    Facility    @relation(fields: [category], references: [id])
+  category      Category
+  @@id([post, category])
+}
+
+model Category {
+  id             Int        @id @default(autoincrement())
+  name           String
+  posts          Post[]
+}
+```
```

Comment:
Maybe add some comments to this schema that this is incorrect so that people don't misunderstand it when only glancing over it:

```suggestion
```prisma
// This schema shows how NOT to define an explicit m-n relation

model Post {
  id            Int         @id @default(autoincrement())
  title         String
  categories    Category[] // this should refer to PostCategories
}

model PostCategories {
  postId        User        @relation(fields: [post], references: [id])
  post          Post
  categoryId    Facility    @relation(fields: [category], references: [id])
  category      Category
  @@id([post, category])
}

model Category {
  id             Int        @id @default(autoincrement())
  name           String
  posts          Post[] // this should refer to PostCategories
}
```
```